components/filter/test/filter-tags.vdiff.js

```diff
@@ -124,7 +124,7 @@ describe('filter-tags', () => {
 			await sendKeys('press', 'ArrowRight');
 			await sendKeys('press', 'Delete');
 
-			await aTimeout('300'); // clearing has a 200ms delay
+			await aTimeout('400'); // clearing has a 200ms delay
```

Comment:
Is this change still needed?

Reply:
what does the comment mean in this context? It seems to be suggesting that it's clearing an existing 200ms delay

Reply:
The comment was already here, I changed the test before but now I've resolved Margaree's previous comments and I don't need to mess with the tests anymore